Sources/FluentUI_iOS/Components/Notification/FluentNotification.swift

```diff
@@ -323,6 +327,13 @@ public struct FluentNotification: View, TokenizedControlView {
 #if os(visionOS)
                 .glassBackgroundEffect(in: RoundedRectangle(cornerRadius: tokenSet[.cornerRadius].float))
 #endif // os(visionOS)
+                .offset(y: -bumpVerticalOffset)
+                .onChange_iOS17(of: state.shouldPerformBump) { shouldBump in
```

Comment:
Just to be sure, there is no way we can use the SwiftUI `animation` modifier to achieve this since we need the animation to be continously moving up and down?